packages/flutter/lib/src/material/tabs.dart

```diff
@@ -31,6 +31,10 @@ const EdgeDims _kTabLabelPadding = const EdgeDims.symmetric(horizontal: 12.0);
 const double _kTabBarScrollDrag = 0.025;
 const Duration _kTabBarScroll = const Duration(milliseconds: 300);
 
+// The scrollOffset (velocity) provided to fling() is pixels/ms, and the
+// tolerance velocity is pixels/sec.
```

Comment:
I'm afraid that the factor of 2 was determined experimentally.  I didn't know what to say...